coops-ui/src/views/AnnualReport.vue

```diff
@@ -34,115 +34,124 @@
-              </header>
-              <ARDate />
-            </section>
 
-            <!-- Registered Office Addresses -->
-            <section>
-              <header>
-                <h2 id="AR-step-2-header">2. Registered Office Addresses
-                  <span class="agm-date">(as of {{ ARFilingYear }} Annual General Meeting)</span>
-                </h2>
-                <p>Verify or change your Registered Office Addresses.</p>
-              </header>
-              <OfficeAddresses
-                :changeButtonDisabled="!allowChange('coa')"
-                :addresses.sync="addresses"
-                :registeredAddress.sync="registeredAddress"
-                :recordsAddress.sync="recordsAddress"
-                @modified="officeModifiedEventHandler($event)"
-                @valid="addressFormValid = $event"
-              />
-            </section>
-
-            <!-- Directors -->
-            <section>
-              <header>
-                <h2 id="AR-step-3-header">3. Directors</h2>
-                <p>Tell us who was elected or appointed and who ceased to be a director at your
-                  {{ ARFilingYear }} AGM.</p>
-              </header>
-              <Directors ref="directorsList"
-                @directorsChange="directorsChange"
-                @directorsFreeChange="directorsFreeChange"
-                @allDirectors="allDirectors=$event"
-                @directorFormValid="directorFormValid=$event"
-                @directorEditAction="directorEditInProgress=$event"
-                :asOfDate="agmDate"
-                :componentEnabled="allowChange('cod')"
-              />
-            </section>
-
-            <!-- Certify -->
-            <section>
-              <header>
-                <h2 id="AR-step-4-header">4. Certify Correct</h2>
-                <p>Enter the name of the current director, officer, or lawyer submitting this Annual Report.</p>
-              </header>
-              <Certify
-                :isCertified.sync="isCertified"
-                :certifiedBy.sync="certifiedBy"
-                :currentDate="currentDate"
-                @valid="certifyFormValid=$event"
-              />
-            </section>
-
-            <!-- Staff Payment -->
-            <section v-if="isRoleStaff && isPayRequired">
-              <header>
-                <h2 id="AR-step-5-header">5. Staff Payment</h2>
-              </header>
-              <StaffPayment
-                :value.sync="routingSlipNumber"
-                @valid="staffPaymentFormValid=$event"
-              />
-            </section>
-
-          </div>
-        </article>
-
-        <aside>
-          <affix relative-element-selector="#annual-report-article" :offset="{ top: 120, bottom: 40 }">
-            <sbc-fee-summary
-              v-bind:filingData="[...filingData]"
-              v-bind:payURL="payAPIURL"
-              @total-fee="totalFee=$event"
-            />
-          </affix>
-        </aside>
```

Comment:
We're going to have some major conflicts here 😢  Wish me luck.

Reply:
I wish you luck!